packages/core/src/messenger.ts

```diff
@@ -60,6 +60,15 @@ export class JoinOrgInviteCompletedMessage extends Message<{ orgName: string; op
     }
 }
 
+export class FailedLoginAttemptMessage extends Message<{ location: string }> {
+    template = "failed-login-attempt";
+
+    get title() {
+        const appName = process.env.PL_APP_NAME;
+        return `${appName ? appName + " " : ""}Failed Login Attempt from ${this.data.location})`;
```

Comment:
I think that changing this to "we prevented a ..." would sound too alarming, but am open to doing it.